sample_files/README.md

```diff
@@ -1,5 +1,6 @@
 [![Build Status](https://travis-ci.org/OCA/${REPO_NAME}.svg?branch=${BRANCH_NAME})](https://travis-ci.org/OCA/${REPO_NAME})
 [![Coverage Status](https://coveralls.io/repos/OCA/${REPO_NAME}/badge.png?branch=${BRANCH_NAME})](https://coveralls.io/r/OCA/${REPO_NAME}?branch=${BRANCH_NAME})
+[![Transifex Status](https://www.transifex.com/projects/p/${REPO_NAME}-${BRANCH_NAME_WITH_DASH}/chart/image_png)](https://www.transifex.com/projects/p/${REPO_NAME}-${BRANCH_NAME_WITH_DASH})
```

Comment:
The graph might look better in the description such as:
```markdown

Translation Status
------------------
[![Transifex Status](https://www.transifex.com/projects/p/OCA-l10n-canada-8-0/chart/image_png)](https://www.transifex.com/projects/p/OCA-l10n-canada-8-0)
```
See: https://github.com/OCA/l10n-canada/blob/7.0/README.md